tcms/testruns/tests/test_permissions.py

```diff
@@ -2,11 +2,13 @@
 # pylint: disable=invalid-name, too-many-ancestors
 
 
-from django.urls import reverse
 from django.utils.translation import ugettext_lazy as _
 
 from tcms.tests import PermissionsTestCase
+from django.urls import reverse
```

Comment:
Why did you rearrange the imports ? Was there a specific reason for that ? 

If not check your pylint output!